src/templates/Footer/index.js

```diff
@@ -9,13 +9,34 @@ import LinkWrapper from 'atoms/LinkWrapper';
 
 import styles from './footer.module.scss';
 
+function renderLinks(linksArr) {
+  if (!linksArr || linksArr.length === 0) return null;
+
+  function renderLink(link) {
+    return (
+      <li className={styles['trust-link']}>
+        <LinkWrapper href={link.href || '#'}>{link.text}</LinkWrapper>
```

Comment:
can we validate in prop types that there is always an `href`? seems counter intuitive to have footer links with dummy `href`s